lib/signs/bus.ex

```diff
@@ -156,17 +162,43 @@ defmodule Signs.Bus do
       end
     end
 
-    [top, bottom] =
+    bridge_audio =
+      if chelsea_bridge && bridge_status_raised?(chelsea_bridge_status, current_time) do
+        case bridge_status_minutes(chelsea_bridge_status, current_time) do
+          minutes when minutes < 2 ->
+            [{@drawbridge_soon, []}, {@drawbridge_soon_spanish, []}]
+
+          minutes ->
+            [
+              {@drawbridge_minutes, [PaEss.Utilities.number_var(minutes, :english)]},
+              {@drawbridge_minutes_spanish, [PaEss.Utilities.number_var(minutes, :spanish)]}
+            ]
+        end
+        |> Enum.map(fn {msg, vars} ->
+          %Content.Audio.BusPredictions{message: {:canned, {msg, vars, :audio_visual}}}
+        end)
+      else
+        []
+      end
```

Comment:
I think we discussed this in an earlier PR and agreed to just get things working before breaking stuff out, but thoughts on starting a separate module for audio related things to start? It's getting a little tougher to read/review the code and keep track of existing/new functionality since there's many things going on in `bus.ex` now